### ISSUE TITLE
Remove "/usr/local/bin/local password"

### ISSUE THREAD

**Issue by vijay-wandb** [wandb/terraform-aws-wandb#267]
Today: wandb-local (0.56) includes the command /usr/local/bin/local password. The command was used in the past when passwords were stored in a htpasswd file in /vol/env. The command is still working. Users can successfully login after is has been executed. Is the application sourcing htpasswd file and database for passwords or why is it still working?

Expectation:  If it was a hack from the past and should not be used, we should remove it entirely.

**Comment by abhinavg6** [comment]
@elainaRenee - This is a fair ask, right? Asking if this is something we could have consultants own at some point, or should it be addressed by us directly?

**Comment by elainaRenee** [comment]
Yes we had spoken about removing this. The code path isn't well maintained anymore and using this might cause unexpected issues.
